commitlint.config.mjs

```diff
@@ -0,0 +1,25 @@
+export default {
+  extends: ['@commitlint/config-conventional'],
```

Comment:
This commitlint rule is base on `conventional commits`

Refs: https://www.conventionalcommits.org/ja/v1.0.0/